src/win/fs.c

```diff
@@ -94,7 +94,7 @@
 
 #define TIME_T_TO_FILETIME(time, filetime_ptr)                              \
   do {                                                                      \
-    uint64_t bigtime = ((int64_t) (time) * 10000000LL) +                    \
+    uint64_t bigtime = ((uint64_t) ((time) * 10000000LL)) +                 \
```

Comment:
At this point `time` is actually a double, not a `time_t`, though the double uses one-second units like `time_t`. So multiplying by a LL or ULL results in a double and so still requires the cast. I'll switch to ULL anyway for clarity.

A Windows FILETIME has units of 100-nanosecond "ticks", that is 10 million per second.